ramp-database/ramp_database/tools/leaderboard.py

```diff
@@ -461,12 +481,23 @@ def get_leaderboard(
             for sub in submissions
         ]
         df = pd.DataFrame(data, columns=columns)
+        if leaderboard_type == "new":
+
+            def create_button(cell_value):
+                return f"<button onclick=\"alert(('You killed ' \
+                    + 'submission {cell_value}!'))\">{cell_value}</button>"
+
+            df["Kill Sub"] = df["submission_id"].apply(
```

Comment:
Again just "Kill" or maybe better "Stop" would be enough. It's in a table of submissions already.